lib/datastore/request.js

```diff
@@ -400,7 +401,7 @@ DatastoreRequest.prototype.insert = function(entities, callback) {
  * //-
  * var query = datastore.createQuery('Lion');
  *
- * datastore.runQuery(query, function(err, entities) {});
+ * datastore.runQuery(query, function(err, entities, info, apiResponse) {});
```

Comment:
It is. I'm not sure if it's better to include all responses, the last response, or no response at all. Returning all would mean returning an array, which might only have a single element most of the time. Any thoughts on which way we should go?